packages/terminal/src/node/shell-terminal-server.spec.ts

```diff
@@ -38,3 +39,45 @@ describe('ShellServer', function (): void {
         expect(await createResult).to.be.greaterThan(-1);
     });
 });
+
+describe('#mergeProcessEnv', function (): void {
+
+    this.timeout(5000);
+    let shellTerminalServer: ShellTerminalServer;
+    const knownVariable = 'PATH';
```

Comment:
Maybe just rename this var `const PATH = 'PATH'`, else it feels weird to hide that string behind `knownVariable`.

Reply:
Done